pipeline/extract.py

```diff
@@ -0,0 +1,32 @@
+import requests
+from bs4 import BeautifulSoup
+import logging
+
+
+def config_log() -> None:
+    """Terminal logs configuration"""
+    logging.basicConfig(
+        format="{asctime} - {levelname} - {message}",
+        style="{",
+        datefmt="%Y-%m-%d %H:%M",
+        level=logging.INFO,
+    )
+
+
+def get_sales_information() -> dict:
+    logging.info("Retrieving Sales Data")
+    sales_url = "https://bandcamp.com/api/salesfeed/1/get_initial"
+    response = requests.get(sales_url)
+    if response.status_code != 200:
+        logging.warning(
+            "Couldn't retrieve Sales Data. Status Code %s".format(response.status_code))
+        return {}
+    sales_data = response.json()
+    logging.info("Sales Data retrieved.")
+
+    return sales_data
+
+
```

Comment:
Looks good! Can you add genre scraping function with beautiful soup. e.g get_genre_from_url()

Reply:
We have collectively decided it makes more sense to leave this to the transform script because:
- Adding genre extraction during the extraction phase would blur the lines between extraction and transformation, making the extract function more complex and harder to maintain.
- Not all records from the API are tracks or albums. If we add genre extraction in the extract script, we'd need to clean and filter the data which would then be repeated in transform anyway